templates/projectcategory.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_id": "{{@ system @}}-projectcategory-collect",
+  "namespaced_identifiers": false,
+  "source": {
+    "operation": "list",
+    "properties": {
+      "datatype": "project/category"
```

Comment:
I would call this property "url_part" or something to not confuse it with out datatype name.

Reply:
I'll let this stay as is for now then. But I get your point too.

Reply:
I agree with you. :)